node/shared/src/snark_job_id.rs

```diff
@@ -23,15 +23,15 @@ pub struct LedgerHashTransitionPasses {
 
 impl std::fmt::Display for LedgerHashTransition {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}->{}", &self.source, &self.target)
+        write!(f, "{}-{}", &self.source, &self.target)
```

Comment:
I used `->` to signify a ledger transition. Do we need to change that?

Reply:
This is much more friendly to URL, when you need to copy-paste it to e.g. `curl`